server/room-battle.ts

```diff
@@ -808,6 +808,10 @@ export class RoomBattle extends RoomGames.RoomGame {
 			} else if (lines[2].startsWith(`|request|`)) {
 				this.rqid++;
 				const request = JSON.parse(lines[2].slice(9));
+				if (request === null) {
+					this.forfeitPlayer(player, 'silent');
```

Comment:
`room-battle` also seems like the wrong place for this. Would this cause problems for people playing via command-line?

Reply:
Command-line simulation is documented (https://github.com/smogon/pokemon-showdown/blob/master/sim/SIMULATOR.md)[here].